recipes-core/packagegroups/packagegroup-ni-extra.bb

```diff
@@ -239,12 +239,16 @@ RDEPENDS_${PN} += "\
 	trace-cmd \
 "
 
-# openembedded-gore/meta/recipes-multimedia
+# openembedded-core/meta/recipes-multimedia
 RDEPENDS_${PN} += "\
 	alsa-tools \
 	alsa-utils \
 	alsa-utils-scripts \
+	ffmpeg \
 	flac \
+	gstreamer1.0-libav \
+	gstreamer1.0-plugins-good \
+	gstreamer1.0-rtsp-server \
```

Comment:
Can the `ffmpeg` and `gstreamer` changes be broken out into their own patchset? I think we need some more time to figure out if we *can* distribute their content, and it doesn't seem to be blocking for the rest of the `gcc 12` patchsets.

Reply:
Split out